virttest/qemu_monitor.py

```diff
@@ -2739,18 +2870,20 @@ def operate_dirty_bitmap(self, operation, node, name, granularity=65536):
         :param name: name of the dirty bitmap to operate
         :param granularity: granularity to track writes with
         """
-        return self._operate_dirty_bitmap(operation, node, name,
+        cmd = "block-dirty-bitmap-%s" % operation
+        if not self._has_command(cmd):
+            cmd += "x-"
+        return self._operate_dirty_bitmap(cmd, node, name,
```

Comment:
It would give a `maximum recursion depth exceeded` error when the `operation` is not supported by the given qemu, which is not clear to users.

Reply:
Uh, sorry.. that's my fault, please ignore it, thanks.